src/components/calcite-button/calcite-button.scss

```diff
@@ -1,3 +1,32 @@
+// theme variables
+// light theme
+:host {
+  --calcite-button-blue: #{$ui-blue};
+  --calcite-button-blue-hover: #{$ui-blue-hover};
+  --calcite-button-blue-pressed: #{$ui-blue-pressed};
+  --calcite-button-red: #{$ui-red};
+  --calcite-button-red-hover: #{$ui-red-hover};
+  --calcite-button-red-pressed: #{$ui-red-pressed};
+  --calcite-button-blue-inline-underline: #{rgba($ui-blue, 0.4)};
+  --calcite-button-red-inline-underline: #{rgba($ui-red, 0.4)};
+  --calcite-button-blue-solid-color: #{$blk-000};
+  --calcite-button-red-solid-color: #{$blk-000};
+}
+
+// dark theme
+:host([theme="dark"]) {
+  --calcite-button-blue: #{$ui-blue-dark};
+  --calcite-button-blue-hover: #{$ui-blue-hover-dark};
+  --calcite-button-blue-pressed: #{$ui-blue-pressed-dark};
+  --calcite-button-red: #{$ui-red-dark};
+  --calcite-button-red-hover: #{$ui-red-hover-dark};
+  --calcite-button-red-pressed: #{$ui-red-pressed-dark};
+  --calcite-button-blue-inline-underline: #{rgba($ui-blue-dark, 0.4)};
+  --calcite-button-red-inline-underline: #{rgba($ui-red-dark, 0.4)};
+  --calcite-button-blue-solid-color: #{$blk-230};
+  --calcite-button-red-solid-color: #{$blk-230};
```

Comment:
$blk-230 passes accessibility for blue and red btns. The lightest we can go to is 240, but 230 looks good.

Reply:
240 would be darker here yeah? so lightest would be 220? These are also against the "current" calcite-color values, so we can see how it works when those are updated?

Reply:
ah, typo. 220 would be the lightest, #2b2b2b. Yeah, sounds good.